src/app/components/play-round/play-round.component.scss

```diff
@@ -77,6 +76,14 @@
   text-shadow: 0 0 0.6em #111;
 }
 
+.marker-name {
+  @include responsive-font-size(2rem, 2.5rem, 3rem);
+}
+
+.nest_guide {
+  @include responsive-font-size(1.1rem, 1.5rem, 2rem);
```

Comment:
Any particular reason this is `1.1rem` rather than `1rem`? Also, we usually don't use underscores for class names.